src/screens/WalletConnect/CallRequest/TransactionRequestContent.js

```diff
@@ -91,37 +97,70 @@ function TransactionRequestContent({ request, onConfirm, onReject }: Props) {
   const chainConfig = chainConfigs[chain];
   const value = wrapBigNumberOrNil(transactionPayload?.amount);
   const symbol = transactionPayload?.symbol;
-  const confirmTitle = !hasNotEnoughGas ? t('button.confirm') : t('label.notEnoughGas');
+  const confirmTitle = !hasNotEnoughGas ? t('button.swipeConfirm') : t('label.notEnoughGas');
   const isConfirmDisabled = isEstimating || hasNotEnoughGas || !!errorMessage;
 
+  const SwiperIconComponent = () => {
+    return (
+      <View style={styles.swiperView}>
+        <Icon name="arrow-right" />
+      </View>
+    );
+  };
+
   return (
     <>
-      <Text color={chainConfig.color}>
-        {title} {t('label.dotSeparator')} {chainConfig.titleShort}
-      </Text>
+      <IconView>
+        <Icon name={chainConfig.iconName} width={64} height={64} style={styles.chainIcon} />
+        <ServiceIcon source={{ uri: iconUrl }} />
+      </IconView>
 
-      <Image source={{ uri: iconUrl }} style={styles.icon} />
+      <Text color={colors.secondaryText} style={styles.body}>
+        {t('walletConnect.requests.transactionRequest.transactionRequest')}
+      </Text>
 
       <LargeTokenValueView value={value} symbol={symbol} style={styles.tokenValue} />
 
+      {!isConfirmDisabled && <TransactionDeploymentWarning chain={chain} style={styles.transactionDeploymentWarning} />}
+
       {!estimationErrorMessage && (
-        <FeeLabel
-          value={fee}
-          assetSymbol={gasSymbol}
-          assetAddress={gasAddress}
-          isLoading={isEstimating}
-          isNotEnough={hasNotEnoughGas}
-          style={styles.fee}
-          chain={chain}
-        />
+        <FeeView>
+          <FeeLabel
+            value={fee}
+            assetSymbol={gasSymbol}
+            assetAddress={gasAddress}
+            isLoading={isEstimating}
+            isNotEnough={hasNotEnoughGas}
+            chain={chain}
+            mode="actual"
+          />
+          <Icon name="info" width={16} height={16} color={colors.buttonTextTitle} style={styles.infoIcon} />
+        </FeeView>
       )}
 
       {!!errorMessage && <ErrorMessage variant="small">{errorMessage}</ErrorMessage>}
 
-      {!isConfirmDisabled && <TransactionDeploymentWarning chain={chain} style={styles.transactionDeploymentWarning} />}
-
-      <Button title={confirmTitle} onPress={handleConfirm} disabled={isConfirmDisabled} style={styles.button} />
-      <Button title={t('button.reject')} onPress={onReject} variant="text-destructive" style={styles.button} />
+      <SwipeButton
```

Comment:
Swipe button in disabled state looks a bit rough:
![image](https://user-images.githubusercontent.com/6368606/128314408-c65743c9-45d5-4784-8869-886f09e097c9.png)

Reply:
@paragramm what will be the design of swiper button in disable state?

Reply:
![button](https://user-images.githubusercontent.com/66996635/128485706-d64ffdf9-80ad-4f31-b1e5-36158978963c.png)

Reply:
@paragramm just a reminder re: above

Reply:
@divyaBacancy there's no disabled state for swiper button. Instead, use a regular disabled button

Reply:
![swipeDisabledBtn](https://user-images.githubusercontent.com/66996635/129743261-d749b3a8-de4e-4d12-b23d-177d8023575f.png)